README.yaml

```diff
@@ -158,7 +158,40 @@ usage: |-
   Detailed information about launch-time configuration is in the [customization](./docs/customization.md)
   document, available from within the shell via `man customization`.
 
+  #### Customizing the installation script
```

Comment:
Should this be in https://github.com/cloudposse/geodesic/blob/8f987c7b3e7ec67449c886dd2a7412bb52d7e796/docs/customization.md instead?